src/service/viewer-impl.js

```diff
@@ -83,6 +83,7 @@ const TRUSTED_VIEWER_HOSTS = [
    * - www.google.cat
    */
   /(^|\.)google\.(com?|[a-z]{2}|com?\.[a-z]{2}|cat)$/,
+  /(^|\.)gmail\.dev$/,
```

Comment:
@ampproject/wg-approvers This allows Gmail's AMP4EMAIL playground to enable `xhrInterceptor` viewer capability. It provides a better (closer to prod) testing experience for email senders.

`isTrustedViewer()` is currently checked when:
- Get/set CID from viewer
- Overriding `referrerUrl` and `viewerUrl`
- Special viewer impression tracking
- Other AMP4EMAIL/AMP4ACTIONS features

Reply:
Can we take this opportunity to get rid of this hard-coded list and instead supply it via config that is controlled by the AMP cache? Just like all the other vars in https://github.com/ampproject/amphtml/blob/master/src/config.js#L45

Reply:
done